components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/style.module.scss

```diff
@@ -80,7 +80,10 @@
 }
 
 .uploadedImages {
-  max-height: 300px;
```

Comment:
maybe this should be shared via a CSS variable?

Reply:
Was not able to share a `css-variable` down through the `untrusted-iframe`